### DISCUSSION TITLE
I can't set the values

### DISCUSSION THREAD

**Discussion by Kelysa** [Profitbase/PowerBI-visuals-Risk#4] [Q&A]
Hello,

When I want to put my columns “impact” and “probability”, it puts them directly in “number of impact” and “number of probability”, whereas my data are numbers.
It's impossible not to summarize my columns ...
<img width="257" alt="Capture" src="https://github.com/user-attachments/assets/06ecddc4-cd1a-4975-9759-f1307bff41f2">

Have you had this problem before?

Thank you in advance.

**Comment by AndersAano** [comment]
Hi @Kelysa,

The these fields should be the values per risk, and should not be summarized. 

Best regards
Anders
